src/node_http2.cc

```diff
@@ -818,7 +818,7 @@ int Http2Session::DoWrite(WriteWrap* req_wrap,
   return 0;
 }
 
-void Http2Session::AllocateSend(size_t recommended, uv_buf_t* buf) {
+void Http2Session::AllocateSend(uv_buf_t* buf) {
```

Comment:
I'm good with removing this. Just fwiw, I included this to follow a pattern used elsewhere (tho slightly differently) 
 e.g. https://github.com/nodejs/node/blob/master/src/tls_wrap.cc#L675